ui/src/app/kvm/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const KVMHeaderNames = {
+  ADD_KVM: "addKVM",
+  COMPOSE_VM: "composeVM",
+  DELETE_KVM: "deleteKVM",
+  REFRESH_KVM: "refreshKVM",
+} as const;
```

Comment:
Yeah, it seems like using normal objects makes it easier on the "vanilla JS" side of things but a bit awkward on the TS side of things.

I think using `as const` is a bit more intuitive, at least for me. Enums are kind of hazy since they function as both a value and a type, not to mention the odd bi-directional data thing that we never make use of. But an object is just an object, and even though the type `ValueOf<typeof Foo>` is long, it's at least clear (again, to me) that the type is saying "only the values of Foo allowed", and there is a clear separation between what's the type and what's the value.

But honestly the main benefit is the improved IntelliSense. 😆